Accesibility branch

Metsanpeitto opened the pull request.

Metsanpeitto committed:
Add ID to url as well

Metsanpeitto committed:
First commit

Metsanpeitto committed:
Change the document name to be more descriptive

TouringIdk commented:
Title :
The document title can be more descriptive, you can try to put your name on it :

Image text alternatives:
All the images of your page have alternative text, that is good users with bad connections or screen readers

Headings:
Your Headings are following a nice hierarchical structure, and you are only using a h1, you are using it correctly

Resize
text: your text have nice size in every screen size

Others:
The page has a linear design, so it can be readable easily on screen reader, have a nice contrast, so will be difficult to lost when reading and have a nice keyboard navigation

The page dont have fast animations, videos or blinking colors, looks nice!

Metsanpeitto committed:
Fix stylelint errors

Metsanpeitto committed:
Link the menu items with sections of the page using ID. Leave class for styling.

Metsanpeitto committed:
Fix html errors

Dipeshtwis reviewed (changes requested):
Hi @Metsanpeitto, 

Great work on making the project. Your comment is professional and also PR title is good.👏 
However, there are some issue which I've mentioned below. Please fix them in order to move forward.

## Required Changes ♻️

- [ ] Your project has failed one contrast color pairs. Here's a [link](https://color.a11y.com/Contrast/) to the contrast validator used. Kindly make the change and update your report. Below is the screenshot as well. 

![Screenshot 2021-06-15 at 23-25-39 Color Contrast Accessibility Validator](https://user-images.githubusercontent.com/22434660/122101038-7f561780-ce31-11eb-8313-6fc6ea5102ce.png)



- [ ] In the PR description section please include a statement pertained to every point checked:

 - If everything was ok, then make a statement about that (e.g. "Page titles - no issues found").
- If issues were found, you should link to the corresponding line of code in the Pull Request changes.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- N.A.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**

-------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Metsanpeitto committed:
Fix color contrast

Metsanpeitto committed:
Fix stylelint errors

Metsanpeitto commented:
After receiving the pull request respond I got to fix the different errors and suggestions that the code reviewer pointed to me.

Those were :
- Contrast color: This error, was pointed out by the code reviewer. It was fixed. 
 You can check it following this link to the line of code: https://github.com/Metsanpeitto/Portfolio_1/pull/9/files#diff-09dd22f6a9132f8d7fbad7b817bf35b4ea0cb738902622cb88d97df4411523dbR33.
:heavy_check_mark: Fixed 

- Title :
The document title can be more descriptive, you can try to put your name on it.
https://github.com/Metsanpeitto/Portfolio_1/pull/9/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R31
 :heavy_check_mark: Fixed 

- Image text alternatives:
All the images of your page have alternative text, that is good users with bad connections or screen readers. 
https://github.com/Metsanpeitto/Portfolio_1/pull/9/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L99
:heavy_check_mark: No issues found.

- Headings:
Your Headings are following a nice hierarchical structure, and you are only using an h1, you are using it correctly.
:heavy_check_mark: No issues found.

- Resize:
 Text: your text has a nice size in every screen size.
:heavy_check_mark:  No issues found.

- Others:
The page has a linear design, so it can be read easily on a screen reader, have a nice contrast, so will be difficult to lose when reading, and have nice keyboard navigation.
:heavy_check_mark: No issues found.

- The page doesn't have fast animations, videos, or blinking colors, looks nice!
:heavy_check_mark: No issues found.
